lib/server.js

```diff
@@ -38,9 +38,13 @@ function Server(universes, port) {
   this.universes = universes || [0x01];
   this.port = port || e131.DEFAULT_PORT;
   this._socket = dgram.createSocket('udp4');
-  this._lastSequenceNumber = 0;
-
+  this._lastSequenceNumber = {};
+	
```

Comment:
Remove the indentation, but leave the extra line. If you check the diff, you will see you have orphan extra spaces there.